sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/sharedconfig/prod.webpack.config.js

```diff
@@ -1,14 +1,6 @@
-var webpack = require("webpack");
 var merge = require("webpack-merge");
 
 var generatedConfig = require("./scalajs.webpack.config");
 var commonConfig = require("./common.webpack.config.js");
-var UglifyJsPlugin = require("uglifyjs-webpack-plugin");
 
-module.exports = merge(generatedConfig, commonConfig, {
-  plugins: [
-    new UglifyJsPlugin({
-      sourceMap: true
-    })
-  ]
-});
+module.exports = merge(generatedConfig, commonConfig);
```

Comment:
Uglify is not needed anymore (code is automatically minified in production mode, and this plugin is not compatible with Webpack 5)

Reply:
Do you need `terser`?

Reply:
From https://webpack.js.org/plugins/terser-webpack-plugin/:
> if you are using webpack v5 or above you do not need to install this plugin. Webpack v5 comes with the latest terser-webpack-plugin out of the box